src/Illuminate/Console/GeneratorCommand.php

```diff
@@ -83,6 +83,8 @@ protected function getPath($name)
 	 */
 	protected function parseName($name)
 	{
+		$name = strtok($name, '@');
```

Comment:
It performs better because it only takes the first part. Explode will return an array of all chunks. Not what we need.